pages/services/kafka/2.5.0-2.1.0/notes/index.md

```diff
@@ -46,11 +46,14 @@ render: mustache
 
 ### Updates
 - Upgrade {{ model.techShortName }} base tech to version 2.1.0. See [{{ model.techShortName }}'s Release Notes](https://kafka.apache.org/21/documentation.html#upgrade_210_notable) more for details.
+- Upgrade Zookeeper Client version to 3.4.13.
 
 ## Version 2.4.0-1.1.1
 
 ### Updates
+- Upgrade {{ model.techShortName }} base tech to version 1.1.1. See [{{ model.techShortName }}'s Release Notes](https://archive.apache.org/dist/kafka/1.1.1/RELEASE_NOTES.html) for details.
```

Comment:
This should be base tech version `2.1.0`, right?

Reply:
It is the update to 2.4.0-1.1.1